proteus/FemTools.py

```diff
@@ -3095,15 +3095,21 @@ def getInverseValuesGlobalExteriorTrace(self,
 """
 
 class ParametricFiniteElementSpace:
-    """
-    Base class for spaces of functions defined by a set of finite
-    elements that are each related to the same reference finite element.
-
-    dim -- global dimension of the space
-    mesh -- a Mesh object, which is a partition of the domain
-    finiteElements -- a dictionary of the FiniteElement objects
-    indexed by element number
-    dofMap -- a DOF
+    """ Base class for finite elements defined on the reference element.
+
+    Parameters
+    ----------
+    referenceFiniteElement : class
```

Comment:
Cool, so this does work.  It does require some special markup.  For the mesh you need to say something like :class:`proteus.MeshTools.Mesh` for example.  The next commit will have the reference included.